.ci/scripts/publish_plugin_pypi.sh

```diff
@@ -12,6 +12,14 @@ cd "$(dirname "$(realpath -e "$0")")"/../..
 
 set -euv
 
+export VERSION=$(http pulp/pulp/api/v3/status/ | jq --arg plugin deb -r '.versions[] | select(.component == $plugin) | .version')
```

Comment:
```suggestion
export VERSION=$(http pulp/pulp/api/v3/status/ | jq --arg plugin pulp_deb -r '.versions[] | select(.component == $plugin) | .version')
```
Assuming this is using the old status api...